cli/toolchain_cmd.go

```diff
@@ -86,15 +67,6 @@ func init() {
 		log.Fatal(err)
 	}
 
-	_, err = c.AddCommand("add",
```

Comment:
Sorry, but what is the reason of sacrificing `toolchain add`? When working on new (or forked) toolchain I'm using `add` a lot..